rfq/oracle.go

```diff
@@ -428,21 +428,6 @@ func NewMockPriceOracle(expiryDelay,
 	}
 }
 
-// NewMockPriceOracleSatPerAsset creates a new mock price oracle with a
-// specified satoshis per asset rate.
-//
-// TODO(ffranr): Remove in favour of NewMockPriceOracle.
-func NewMockPriceOracleSatPerAsset(expiryDelay uint64,
```

Comment:
Hmm, so this was actually broken in 4e291e5e and we didn't realize back then. But the meaning between `rfqCfg.MockOracleAssetsPerBTC` and `rfqCfg.MockOracleSatsPerAsset` is different, so one of these functions is now incorrect, probably this one (since a rate is assets per BTC, so we'd need to convert things from sats per asset to that rate, which will be imprecise).
Or we completely remove the whole config flag for the sats per asset way of configuring the mock oracle.

Reply:
I'll leave the old constructor in and I'll fix the bug. We need to convert sats per asset to asset per btc correctly.

Reply:
I've added a TODO in this PR to fix this oversight.